examples/sparse_2of4_quantization_fp8/README.md

```diff
@@ -106,6 +106,16 @@ Output Directories:
 - Without FP8: `Meta-Llama-3-8B-Instruct-2of4-sparse`
 - With FP8: `Meta-Llama-3-8B-Instruct-2of4-W8A8-FP8-Dynamic-Per-Token`
 
+Save Model on disk without sparse_compression:
```

Comment:
Can we update this to be a bit more descriptive and fix the grammar
We should also make this a subheading 
We should also add a note: 

`This will compress the model using the quantization compressor however, instead of using the optimal sparsity compressor, the dense sparsity compressor will be used`

Reply:
Updated!